cli/src/main.rs

```diff
@@ -56,6 +74,16 @@ async fn main() -> Result<(), color_eyre::Report> {
                 .any(|group| group.contains(&arg.as_str())))
         {
             print_help(&styling)?;
+            // WORKAROUND for #2212: because of how `color_eyre`
+            // works, we need to install the hook before creating any
+            // instance of `eyre::Report`, otherwise the `eyre`
+            // default reporting hook is going to be installed
+            // automatically.
+
+            // This results in a nasty repetition of the
+            // `color_eyre::install().unwrap()` pattern, which is the
+            // lesser of two evils
+            color_eyre::install().expect("CRITICAL");
```

Comment:
This is a bit confusing to me as we call the hook inside this function and a few times in `.map_err(..)` invocations. Is it really needed or would moving the call to the top of `main()` help?